service/controller/resource/deployment/desired_test.go

```diff
@@ -319,7 +445,7 @@ func Test_Resource_Deployment_GetDesiredState(t *testing.T) {
 
 		certsSearcher, err = certs.NewSearcher(c)
 		if err != nil {
-			t.Fatal("expected", nil, "got", err)
+			return nil, err
```

Comment:
We always need to mask the error when we return it. There are other spots where this should be applied.
```suggestion
			return nil, microerror.Mask(error)
```

Reply:
Even in test helper functions?

Reply:
Updated

Reply:
It is needed for stack traces. Not sure if it's super important here, but it's good to be consistent.